python/Makefile

```diff
@@ -31,18 +31,19 @@ gen:
 	for f in ${PROTO_FOLDERS}; do \
 		poetry run python -m grpc_tools.protoc \
 			-I"$(GIT_TOP_DIR)" \
-			-I"$(PROTOBUF_INCLUDE_DIR)" \
-			-I"${GOPATH}/src/github.com/googleapis/googleapis" \
+			-I"$(GIT_TOP_DIR)/proto/external/googleapis/googleapis/83e756a66b80b072bd234abcfe89edf459090974" \
 			--python_out=$(GIT_TOP_DIR)/python/src \
 			$$f/*.proto; \
+	done
+
+	for f in ${PROTO_SERVICE_FOLDERS}; do \
```

Comment:
We didn't notice because the `test-py` job doesn't return an error, but there were many error logs while generating the proto files. So I split the loop to generate the correct files and deleted the unnecessary include `PROTOBUF_INCLUDE_DIR` directory.

![Screen Shot 2022-10-12 at 23 00 47](https://user-images.githubusercontent.com/2486691/195363745-ec34e10e-2dcf-4535-b80e-ab69a683ac19.png)